src/amo/components/PermissionsCard/styles.scss

```diff
@@ -0,0 +1,29 @@
+.PermissionCard-learn-more {
+  margin: 12px auto 0;
+  width: 100%;
+
+  .Icon {
+    margin-left: 10px;
+  }
+}
+
+.PermissionsCard-subhead {
+  color: #737373;
+}
+
+.PermissionsCard-list {
+  list-style: none;
+  margin: 10px 0 0;
+  padding: 0;
+
+  li {
+    display: block;
+    margin-left: 10px;
+    margin-top: 10px;
+    vertical-align: middle;
+
+    .Icon {
+      margin-right: 10px;
```

Comment:
The pixel distance I see in Sketch is `16px`. Also, generally, we go with multiples of 4 but this isn't a hard and fast rule. There are a lot of `10px` measurements on the site left over from early days. This kind of distance is more like `8px` or `12px` in the latest mocks.

Reply:
Hrm, again it looks different between the Screens and the Symbols pages. I guess I'll just go with 12 for now, as 16 looks like too much to me.